client/my-sites/customer-home/cards/tasks/promote-post/index.js

```diff
@@ -10,13 +11,22 @@ import {
 } from 'calypso/lib/promote-post';
 import { TASK_PROMOTE_POST } from 'calypso/my-sites/customer-home/cards/constants';
 import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
-import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getSiteOption } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
 const PromotePost = () => {
 	const translate = useTranslate();
 	const showPromotePost = usePromoteWidget() === PromoteWidgetStatus.ENABLED;
 	const dispatch = useDispatch();
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const adminInterface = useSelector( ( state ) =>
+		getSiteOption( state, siteId, 'wpcom_admin_interface' )
+	);
+	const advertisingUrl =
+		adminInterface === 'wp-admin' && isEnabled( 'layout/dotcom-nav-redesign' )
+			? `https://${ selectedSiteSlug }/wp-admin/tools.php?page=advertising`
```

Comment:
It would be better to use `getSiteAdminUrl` to get the admin URL 🙂

Reply:
Good call. Thanks!